src/style.css

```diff
@@ -0,0 +1,24 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  display: flex;
+  margin-right: 4px;
+  background: url("images/star.svg") center no-repeat;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0),
```

Comment:
```suggestion
```
The :nth-child(-n + 0) selector will never match any elements because it selects elements that are the first child of their parent and also have no preceding siblings, which is impossible. You can remove the .stars--0 style completely as it doesn't affect the styling.